content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -31,7 +32,8 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
```

Comment:
We should figure out how the minimal profile fits into multicluster. If it is unusable with multiple clusters we maybe should say so here?

Reply:
That's outside of the scope of this PR. Here, I'm just introducing the new terminology.